src/globus_sdk/services/transfer/client.py

```diff
@@ -2177,14 +2182,16 @@ def endpoint_manager_cancel_status(
         <https://docs.globus.org/api/transfer/advanced_endpoint_management/#get_cancel_status_by_id>`_
         in the REST documentation for details.
         """
+        # REVIEW:
+        admin_cancel_id = utils.safe_stringify(admin_cancel_id)
```

Comment:
I'm surprised that you were able to write it like this, rather than needing a new var name.

If you want review on the proper types for `admin_cancel_id` in the first place, I think we should ask for @aaschaer to weigh in. I'm pretty sure it's a UUID, but it would be good to confirm.

Reply:
Yes, admin_cancel_id must be a UUID